mitmproxy/addons/save.py

```diff
@@ -54,6 +108,16 @@ def configure(self, updated):
                     raise exceptions.OptionsError(str(e)) from e
             else:
                 self.filt = None
+        if "save_stream_period" in updated:
+            if ctx.options.save_stream_period:
+                if ctx.options.save_stream_period == 'hour':
```

Comment:
Instead of two hard-coded options, what about save_stream_hours as the number of hours for rotation?   24 for a day, or 0.25 for every fifteen minutes? An hour of video might be too much in some scenarios?

Reply:
That would work too, although it would get tedious for 10 minute intervals. In this case I would suggest to go with `save_stream_minutes` or `save_stream_seconds` to keep the supplied number an integer. We could discuss if sub-minute resolution is really a use case but I would be fine with seconds.

Reply:
There surely is some standard way in Python to have human readable interval format (like CRON but in nice). We do the same in other options, e.g. you can specify `3m` in `stream_large_bodies` for 3 megabytes. I'm sure we're not the first that want something like `24h` oder `15min` and have it parse into a duration/interval.